MonoXAnalysis/cfg/run_monojet_cfg.py

```diff
@@ -8,6 +8,8 @@
 # Load all analyzers
 from CMGTools.MonoXAnalysis.analyzers.dmCore_modules_cff import * 
 from PhysicsTools.HeppyCore.framework.heppy_loop import getHeppyOption
+from CMGTools.HToZZ4L.tools.configTools import * 
+
```

Comment:
The AAA has been moved here, you need to change
https://github.com/CERN-PH-CMG/cmgtools-lite/pull/25